K2Bridge.Tests.End2End/TestElasticClient.cs

```diff
@@ -182,6 +218,9 @@ public async Task<JToken> MSearch(string indexName, string jsonQueryFile, bool v
             // Normalize aggregate value (double) with fixed number of decimal
             NormalizeAggregateValue(result, "responses[*].aggregations..value");
 
+            // make a Math.Round on all floats values
+            RoundFloats(result, "responses[*].aggregations", roundingFloats);
+
```

Comment:
Ok. Nice, it means NormalizeAggregateValue is probably not needed any longer. You can remove it.

Reply:
Forget. We will see that in another PR.